### ISSUE TITLE
Question: Higher Resolution

### ISSUE THREAD

**Issue by yumfab-eeis** [AaronLeong/BigGAN-pytorch#3]
Have you tried to generate higher resolution images? (512/1024/higher)

**Comment by AaronLeong** [comment]
Generate  256 resolution image(batch size 64) will use about 10G memory per GPU,  so I don't have enough machines to try ...

**Comment by Ianmcmill** [comment]
Would decreasing the batch size to say 4 during training also decrease "quality" of resulting images? Or would just take longer but give the same output as a batch size of 64?
And let's say I have several servers with several GPUs with 24gb (respectively 48gb (purely hypothetical of course), would increasing the batch size to 128/256 result in higher quality/accuracy?
